test/e2e_node/runner/remote/run_remote.go

```diff
@@ -125,12 +126,31 @@ type internalGCEImage struct {
 	tests    []string
 }
 
+// parseFlags parse subcommands and flags
+func parseFlags() {
+	if len(os.Args) <= 1 {
+		glog.Fatalf("Too few flags specified: %v", os.Args)
+	}
+	// Parse subcommand.
+	subcommand := os.Args[1]
+	switch subcommand {
+	// TODO: Add subcommand for node soaking, node conformance, cri validation.
+	default:
+		// Use node e2e suite by default if no subcommand is specified.
+		suite = remote.InitNodeE2ERemote()
+	}
+	// Parse test flags.
+	flag.CommandLine.Parse(os.Args[2:])
```

Comment:
Parse the flags first, then use `flag.Args` instead of `os.Args`. That way it doesn't matter if the flags are specified before or after the positional argument.